app/models/server/models/Users.js

```diff
@@ -1622,6 +1623,43 @@ Find users to send a message by email if:
 			},
 		});
 	}
+
+	// Voip functions
```

Comment:
Since our end goal on backend is to move all query method to use the raw model instead of this traditional model, I'd recommened moving all these methods to the respective raw User model i.e. [this one](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/models/server/raw/Users.js)

Reply:
@murtaza98  How do we do it? REST API layer seem to use the traditional model. Can you please point me to how to use this?

Reply:
I tried that but didnt work just by moving the APIs there.

Reply:
@amolghode1981 you'll have to modify the rest layer API's to use the new raw models instead of the traditional collection. You can import the raw collection there using 
`import { Users } from '../../../../models/server/raw';` instead of `import { Users } from '../../../../models/server/index';`

Sidenote: Notice in the new file which you're importing above, [here](https://github.com/RocketChat/Rocket.Chat/blob/f1ce17dd71060a6faecde14e73f047aaad72afd0/app/models/server/raw/index.ts#L76) we're combining the Raw and traditional models, so this way using the Raw model, you'll be able to use all the traditional model method, but you won't get typescript definitions for them. If you want typescript definitions, then you'll have to declare them inside Raw models

Reply:
Fixed.